### PR TITLE
Adds new footer items for feedback and github

### PR DESCRIPTION
**Pull request by jenniferthibault**
## Changes
Adds contact copy for email and github, as determined in https://github.com/18F/openFEC-web-app/issues/866#issuecomment-154172692

## Notes: 
Goes with https://github.com/18F/fec-style/pull/174

I can’t seem to get the image code quite right here, and would appreciate help!

Might have to add an image class afterwards for image positioning?

## Review: 
@noahmanger

### CODE REVIEW

File: templates/layouts/main.html [hunk 1]
```diff
@@ -78,7 +78,9 @@
       <p class="address__title">Federal Election Commission</p>
       <p>999 E Street, NW, Washington, DC 20463</p>
       <p>(800) 424-9530</p>
-      <p><a href="mailto:{{ contact_email }}">{{ contact_email }}</a></p>
+      <p class="address__subtitle">betaFEC feedback</p> 
+      <p><a href="mailto:{{ contact_email }}">{{ contact_email }}</a><img src="img/i-email--inverse.svg" alt="Icon of email being sent"></p>
+      <p><a href="https://github.com/18F/fec">github.com/18F/FEC</a><img src="img/i-github--inverse.svg" alt="Icon of GitHub logo"></p>
```
Comment (noahmanger) [inline templates/layouts/main.html]: Let's make the link text 'GitHub repository' like it is on the Contact us page.

Comment (noahmanger) [inline templates/layouts/main.html]: I made this change myself.

Comment (jenniferthibault) [inline templates/layouts/main.html]: Pinging @emileighoutlaw for review on the GitHub footer link text, as it was her call to use the same format as 18F's footer link in https://github.com/18F/openFEC-web-app/issues/866#issuecomment-151214687 . I definitely see the case for making both the footer and contact page match, whichever format we go with.

**Comment by emileighoutlaw** [comment]
I think it's okay to go with `GitHub repository` for now. I'm going to keep looking for more information on this microcopy question. I'll open a new issue later, if I want to change (but a hearty yes to consistency with Contact Us page and footer).

**Pull request merged**